.github/pull_request_template.md

```diff
@@ -35,9 +35,20 @@ Please mark with X for applicable areas.
 | Other                      | [ ] |
 
 
+**Testing done**:
+
+<!--
+
+Please describe how are the changes tested? You can add supporting information, E.g. screenshots, logs etc.
+
+-->
+
+
+**Miscellaneous**
```

Comment:
Not sure if we need this really. Let's keep it simple and not have it unless there is a need. In general Miscellaneous comments can be regular GH comments.